src/Components/Movie/Movie.scss

```diff
@@ -0,0 +1,4 @@
+.movie-poster-img {
+  height: 100px;
+  width: 100px;
```

Comment:
Once we get to styling, let's consider switching the sizing to vh and vw to help with responsiveness later on.  This is a great placeholder for now, so the images don't take over the screen.